library-aux/src/main/scala/scala/runtime/Statics.scala

```diff
@@ -34,56 +34,35 @@ object Statics {
   }
 
   def longHash(lv: Long): Int = {
-    lv.toInt
-    /*
-    val iv = lv.toInt | 0
+    val iv = lv.toInt
     if (iv == lv) iv
-    else (lv ^ (lv >>> 32)).toInt | 0
-    */
+    else iv ^ (lv >>> 32).toInt
```

Comment:
I was trying to avoid re fetching `lo` from memory. But maybe that's a tiny bit overkill ^^